build.gradle.kts

```diff
@@ -12,20 +12,24 @@ plugins {
   `maven-publish`
   id("org.jetbrains.dokka") version "1.9.0"
   id("org.jetbrains.kotlinx.kover") version "0.7.3"
+  signing
   idea
+  id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
 }
 
 repositories {
   mavenCentral()
 }
 
 dependencies {
-  detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.23.1")
+  api(project(":common"))
+  api(project(":credentials"))
+  api(project(":crypto"))
+  api(project(":dids"))
 }
 
 allprojects {
-  version = "0.0.9"
-  group = "web5"
+  group = "xyz.block"
```

Comment:
wao, fancy group name. is this to be the groupId for tbdex-kt as well?

Reply:
There's a convo in https://github.com/TBD54566975/open-source-programs/issues/95

But I don't know if that includes tbdex-kt. A decision will need to be made.